addons/script-snap/userscript.js

```diff
@@ -0,0 +1,16 @@
+export default async function ({ addon, global, console }) {
+  const Blockly = await addon.tab.traps.getBlockly();
+  const workspace = await Blockly.getMainWorkspace();
```

Comment:
Haven't tried this locally but I'm guessing that this won't work if the user exits the editor and opens it again. Perhaps do something like https://github.com/ScratchAddons/ScratchAddons/blob/master/addons/editor-sounds/userscript.js